tdrs-backend/tdpservice/users/api/authorization_check.py

```diff
@@ -14,18 +16,16 @@ class AuthorizationCheck(APIView):
 
     query_string = False
     pattern_name = "authorization-check"
+    permission_classes = [AllowAny]
```

Comment:
Add special permission for the authorization check to allow requests from any user. The response indicates the users authorization.